content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/utxo-advanced-ledger-extensions/ledger-extensions-api-reference.md

```diff
@@ -368,11 +368,11 @@ An ownable state can be designed by implementing the `OwnableState` interface:
 class ExampleOwnableState(private val owner: PublicKey) : OwnableState {
 
     override fun getOwner(): PublicKey {
-    return owner
+        return owner
     }
 
     override fun getParticipants(): List<PublicKey> {
-    return listOf(getOwner())
+        return listOf(getOwner())
     }
 }
```

Comment:
This is indented with 4 spaces - while most of the code in the file until now is indented with 2 spaces.
Please be consistent.